src/editor.js

```diff
@@ -603,6 +603,19 @@ export class OdooEditor {
         } while (fakeEl.parentNode);
     }
 
+    _insertFontAwesome(fa = 'star') {
+        const sel = document.defaultView.getSelection();
+        if (!sel.isCollapsed) {
+            this.deleteRange(sel);
+        }
+        // We need to add a &#8203;to ensure the insertHTML command is not ignored
+        if (document.execCommand('insertHTML', false, '<i class="far fa-' + fa + '">&#8203;</i>')) {
+            const node = findNode(closestPath(sel.focusNode), node => node.tagName === 'I');
```

Comment:
This is equivalent to
```js
const focusElement = sel.focusNode.nodeType === Node.TEXT_NODE ? sel.focusNode.parentNode : sel.focusNode;
const node = focusElement.closest('i');
```
We should have an util for that `focusElement` so we could do:
```js
const node = closestElement(sel.focusNode).closest('i');
```
... or add `closest` to the prototype of `TextNode` ;-)

Reply:
👌